apps/imagekit/package.json

```
@@ -0,0 +1,71 @@
+{
+  "name": "imagekit-contentful",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@contentful/app-sdk": "^4.17.1",
+    "@contentful/f36-components": "^4.4.1",
+    "@contentful/react-apps-toolkit": "^1.2.16",
+    "@dnd-kit/core": "^6.3.1",
+    "@dnd-kit/sortable": "^8.0.0",
+    "@dnd-kit/utilities": "^3.2.2",
+    "@emotion/css": "^11.13.5",
+    "@emotion/react": "^11.14.0",
+    "@imagekit/react": "^5.0.1",
+    "file-size": "^1.0.0",
+    "imagekit-media-library-widget": "^2.1.1",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "lodash": "^4.17.21"
+  },
+  "devDependencies": {
+    "@contentful/app-scripts": "^2.3.0",
+    "@tsconfig/vite-react": "^3.4.0",
+    "@types/file-size": "^1.0.3",
+    "@types/react": "^18.3.12",
+    "@types/react-dom": "^18.3.1",
+    "@typescript-eslint/eslint-plugin": "^8.16.0",
+    "@typescript-eslint/parser": "^8.16.0",
+    "@vitejs/plugin-react": "^4.3.4",
+    "cross-env": "7.0.3",
+    "eslint": "^9.27.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-prettier": "^5.2.1",
+    "eslint-plugin-react": "^7.37.2",
+    "eslint-plugin-react-hooks": "^5.2.0",
+    "prettier": "^3.4.1",
+    "typescript": "^5.7.2",
+    "vite": "^6.2.3",
+    "vitest": "^3.2.4",
+    "@testing-library/react": "^16.3.0",
+    "@testing-library/user-event": "^14.6.1",
+    "happy-dom": "^15.11.7",
+    "@types/lodash": "^4.17.20"
+  },
+  "scripts": {
+    "start": "vite",
+    "install-ci": "npm ci",
+    "build": "tsc && vite build",
+    "test": "vitest",
+    "lint": "eslint src",
+    "dev": "vite",
+    "preview": "vite preview",
+    "create-app-definition": "contentful-app-scripts create-app-definition",
+    "add-locations": "contentful-app-scripts add-locations",
+    "upload": "contentful-app-scripts upload --bundle-dir ./build",
+    "upload-ci": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id $CONTENTFUL_ORG_ID --definition-id $CONTENTFUL_APP_DEF_ID --token $CONTENTFUL_ACCESS_TOKEN"
+  },
```

```suggestion
  "scripts": {
    "start": "vite",
    "install-ci": "npm ci",
    "build": "tsc && vite build",
    "test": "vitest",
    "lint": "eslint src",
    "dev": "vite",
    "preview": "vite preview",
  },
```

just going to have you remove deploy scripts while this goes through the review process, we'll add them for you back in later

---

makes sense, let me just remove them real quick